traffic_ops/testing/api/v3/deliveryserviceservers_test.go

```diff
@@ -111,17 +111,17 @@ func TryToRemoveLastServerInDeliveryService(t *testing.T) {
 
 	_, _, err = TOSession.DeleteDeliveryServiceServer(*ds.ID, *server.ID)
 	if err == nil {
-		t.Error("Didn't get expected error trying to remove the only server assigned to a Delivery Service")
+		t.Logf("Didn't get expected error trying to remove the only server assigned to a Delivery Service")
```

Comment:
Why did you swap these errors to logs and vice versa? I think the logic was correct before.

Reply:
Logic wasn't making sense to me but I have swapped them back to work on later.